assets/components/stripePopUpButton/stripePopUpButton.jsx

```diff
@@ -38,7 +38,7 @@ const StripePopUpButton = (props: PropTypes) => {
     <button
       className="component-stripe-pop-up-button"
       onClick={stripeClick}
-    >Contribute with debit/credit card <Svg svgName="credit-card" /></button>
+    >Pay with debit/credit card <Svg svgName="credit-card" /></button>
```

Comment:
Probably, in order to make it more generic, we should read the button's text from an argument.